perses/dispersed/utils.py

```diff
@@ -20,7 +19,6 @@
 import dask.distributed as distributed
 from scipy.special import logsumexp
 import openmmtools.cache as cache
```

Comment:
This import is safe now with the changes in openmmtools, right? or will importing this create a global context cache?

Reply:
Importing will create a global context cache, but it should be a singleton, I believe.